content/7-known-issues/_index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Known Issues
-weight: 10
+title: 7. Known Issues
```

Comment:
Use `<pre>...`?

Reply:
Fixed in https://github.com/coreruleset/documentation/pull/137/commits/53a0433399d443048850bf3b8de34a4eed9b087b